operator/src/main/java/org/keycloak/operator/controllers/KeycloakSharedCsvMetadata.java

```diff
@@ -91,7 +91,8 @@
             "    \"spec\": {\n" +
             "      \"instances\": 1,\n" +
             "      \"hostname\": \"example.org\",\n" +
-            "      \"tlsSecret\": \"my-tls-secret\"\n" +
+            "      \"http\":\n" +
+            "        \"tlsSecret\": \"my-tls-secret\"\n" +
```

Comment:
Probably something like the following might work

```suggestion
            "      \"http\":\n {" +
            "        \"tlsSecret\": \"my-tls-secret\" } \n" +
```

Reply:
Ah, yeah that makes sense - I was building off the immediately preceding section, but you're right, this is a subdict - very confusing how this is structured, I'm not surprised it got missed initially!